### ISSUE TITLE
Warlock Issues.

### ISSUE THREAD

**Issue by Eradicatepls** [gamefreedomgit/Frostmourne#1756]
Some random things i've noticed over the last days 👍  

- Soulstone not depleting when used, acting like there is no charge anymore. 
- All trinket's are seeming to randomly proc on Life Tap use, including usual Tailoring proc, and dying curse, but now even Embrace of the spider, and Emblem trinket are proccing. Which is really not QoL oriented. Only Stackable trinket should be added in the available proc data base.

**Comment by sputnikk** [comment]
soulstone not depleting might be a ulduar test only thingy since they added the consumables never get used script. try it in naxx or outside world see if it happens there too

**Comment by Eradicatepls** [comment]
Happens both PTR and Live raid.

**Comment by Manpower99** [comment]
soulstone part is only in Ulduar, as for the Procs, players asked for it to work blizzlike so there is it.

**Comment by Eradicatepls** [comment]
Yeah soulstone is working normally outside of ulduar, my bad. But for trinket ? Who asked for that ?

**Comment by Piootrek** [comment]
https://github.com/gamefreedomgit/Frostmourne/issues/1330
I'm waiting for proof of which trinkets should be procced and which shouldn't.

**Comment by Eradicatepls** [comment]
What proof you want, we're asking for a QoL Change, it's either you accept it and only tolerate stacking trinket to get prestack by Life tap or you don't accept this Qol and you remove all chance to proc any trinket from it. I should be procing on harmfull spells it's specified in the tooltip. 

In Classic Wotlk we could prestack illu without any chance of proccing other trinkets which is not supposed to be possible in theory.
ATM You have to remove reequip trinket and cloak to trigger their 30S ICD then prestack and launch a "/dbmpull 30" to get Trinket/Tailoring back at start. With this change you can much easier control it and don't force a pull timer around you.

**Comment by Manpower99** [comment]
with life tap majority of warlocks wanted to pre stack illu, which as it seems blizzlike same as proccing trinkets and cloaks. 
not even a QoL. perhaps if you wish for it to be only stacking illu maybe it a suggestion on the whitemane discord instead of a bug report.

**Comment by Eradicatepls** [comment]
I don't know abouthttps://github.com/gamefreedomgit/Frostmourne/issues/1330 not my issue report. I don't think on blizzlike you could proc trinkets from random Life Tap, though i might be wrong. Hard to find real footage of that. But even if it did it would be stupid.
